### ISSUE TITLE
java.lang.NoClassDefFoundError: io/gatling/core/check/DefaultFindCheckBuilder

### ISSUE THREAD

**Issue by LinkJerome** [Tinkoff/gatling-jdbc-plugin#28]
Hi!

I'm trying to do some database performance tests with your library, but I've an error.
Maybe I did something wrong.
Here's my configuration :

My pom.xml dependencies : 
```xml
<dependency>
	<groupId>io.gatling.highcharts</groupId>
	<artifactId>gatling-charts-highcharts</artifactId>
	<version>3.7.3</version>
	<scope>test</scope>
</dependency>
<dependency>
	<groupId>io.gatling</groupId>
	<artifactId>gatling-app</artifactId>
	<version>3.7.3</version>
</dependency>
<dependency>
	<groupId>io.gatling</groupId>
	<artifactId>gatling-recorder</artifactId>
	<version>3.7.3</version>
</dependency>
<dependency>
	<groupId>io.gatling</groupId>
	<artifactId>gatling-jdbc</artifactId>
	<version>3.7.3</version>
</dependency>
<dependency>
	<groupId>ru.tinkoff</groupId>
	<artifactId>gatling-jdbc-plugin_2.13</artifactId>
	<version>0.5.0</version>
</dependency>
```

My Scala class :

```scala
class TestJDBCFeeder1UserSimulation extends Simulation {

  val dataBase: JdbcProtocolBuilder = DB
    .url("jdbc:postgresql://...:25060/defaultdb")
    .username("...")
    .password("...")

  val scn: ScenarioBuilder = scenario("1 user Select all client")
    .exec(jdbc("select client").query("SELECT * FROM client").check(simpleCheck(_.nonEmpty)))

  setUp(
    scn.inject(atOnceUsers(1)),
  ).protocols(dataBase)
    .maxDuration(60)
}
```

The error :

```logs
20:46:18.744 [main][ERROR][Gatling.scala:90] i.g.a.Gatling$ - Run crashed
java.lang.ClassNotFoundException: io.gatling.core.check.DefaultFindCheckBuilder
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 21 common frames omitted
Wrapped by: java.lang.NoClassDefFoundError: io/gatling/core/check/DefaultFindCheckBuilder
	at ru.tinkoff.load.jdbc.check.JdbcCheckSupport.$init$(JdbcCheckSupport.scala:32)
	at ru.tinkoff.load.jdbc.Predef$.<clinit>(Predef.scala:3)
	at simulations.test.TestJDBCFeeder1UserSimulation.<init>(TestJDBCFeeder1UserSimulation.scala:33)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.lang.Class.newInstance(Class.java:584)
	at io.gatling.app.SimulationClass$Scala.params(SimulationClass.scala:31)
	at io.gatling.app.Runner.run0(Runner.scala:62)
	at io.gatling.app.Runner.run(Runner.scala:49)
	at io.gatling.app.Gatling$.start(Gatling.scala:87)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:49)
	at io.gatling.app.Gatling$.main(Gatling.scala:38)
	at io.gatling.app.Gatling.main(Gatling.scala)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.gatling.mojo.MainWithArgsInFile.runMain(MainWithArgsInFile.java:53)
	at io.gatling.mojo.MainWithArgsInFile.main(MainWithArgsInFile.java:34)
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.gatling.mojo.MainWithArgsInFile.runMain(MainWithArgsInFile.java:53)
	at io.gatling.mojo.MainWithArgsInFile.main(MainWithArgsInFile.java:34)
Caused by: java.lang.NoClassDefFoundError: io/gatling/core/check/DefaultFindCheckBuilder
	at ru.tinkoff.load.jdbc.check.JdbcCheckSupport.$init$(JdbcCheckSupport.scala:32)
	at ru.tinkoff.load.jdbc.Predef$.<clinit>(Predef.scala:3)
	at simulations.test.TestJDBCFeeder1UserSimulation.<init>(TestJDBCFeeder1UserSimulation.scala:33)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.lang.Class.newInstance(Class.java:584)
	at io.gatling.app.SimulationClass$Scala.params(SimulationClass.scala:31)
	at io.gatling.app.Runner.run0(Runner.scala:62)
	at io.gatling.app.Runner.run(Runner.scala:49)
	at io.gatling.app.Gatling$.start(Gatling.scala:87)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:49)
	at io.gatling.app.Gatling$.main(Gatling.scala:38)
	at io.gatling.app.Gatling.main(Gatling.scala)
	... 6 more
Caused by: java.lang.ClassNotFoundException: io.gatling.core.check.DefaultFindCheckBuilder
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 21 more
```

Do you have any idea?

Thanks.

**Comment by jigarkhwar** [comment]
Hello! 
release 0.5.0 is not compatible with Gatling 3.7.x. reason https://github.com/gatling/gatling/issues/4195. We are waiting fix in 3.8.x
Could you try downgrade Gatling’s version?

**Comment by LinkJerome** [comment]
I had to downgrade to the 3.6.1 version, thank you for the tip !
